prime-router/metadata/fhir_mapping/hl7/datatypes/PT/Meta.yml

```diff
@@ -0,0 +1,26 @@
+tag:
+  generateList: true
+  expressionType: nested
+  vars:
+    pt1: STRING, PT.1
+    pt2: STRING, PT.2
+  condition: $pt1 NOT_NULL || $pt2  NOT_NULL
+  expressions:
+    - expressionType: nested
+      condition: $pt1 NOT_NULL
+      expressionsMap:
+        code:
+          valueOf: PT.1
```

Comment:
The inventory actually specifies a valueset mapping for these, but this is not enforced in the FHIR code.  There are many places where we rely on these values be set (filters) and hardcoded spots in the kotlin code where we the field that I opted to not do that translation

Reply:
Good call, I'll drop a comment on the ticket, but since TX is mapping to a primitive there is no work there